.travis.yml

```diff
@@ -3,10 +3,11 @@ python:
     - "2.7"
 install:
     - pip install requests junit_xml beaker-client mock
-    - pip install pylint pycodestyle
+    - pip install pylint pycodestyle flake8
+before_script:
+    - flake8 --show-source .
 script:
     - python -m unittest discover tests
     - pylint tests
-    - pycodestyle .
```

Comment:
Does flake8 include pycodestyle tests, all of them?

Reply:
Yes. @agustinhenze pointed that out to me in another PR. Hooray for less code! :smile:

Reply:
Right, then we'll need to remove `pycodestyle` from [CODING.md](https://github.com/RH-FMK/meta/blob/master/CODING.md), once we switch sktm to flake8.